.bazelrc

```diff
@@ -11,6 +11,9 @@ import %workspace%/.aspect/bazelrc/performance.bazelrc
 # We have some empty globs in rules_js
 common --noincompatible_disallow_empty_glob
 
+# Disable lockfile for now. It is unstable.
```

Comment:
Again link to a bug so we know when to remove

Reply:
There are a number of them right now. Guess I can just list all of the ones outstanding.